common/os_calls.c

```diff
@@ -1692,12 +1692,7 @@ int
 g_set_wait_obj(tintptr obj)
 {
 #ifdef _WIN32
-    if (obj == 0)
-    {
-        return 0;
-    }
-    SetEvent((HANDLE)obj);
-    return 0;
+#error "Win32 is no longer supported."
```

Comment:
That seems like a reasonable thing to do, given recent conversations. Alternatively, move this one to a global place at the top of the file, and over time we can remove the others - you don't need to do this right now. What do you think?

Reply:
Windows builds are already broken. By doing this, I explicitly say we no longer intend to support it to make it crystal clear.

I don't want to move it just so we can remove it later. The refactor I did breaks Windows builds explicitly, so I should be explicit about that.

Trying not to make work for later!